crates/rspack_binding_options/src/options/raw_optimization.rs

```diff
@@ -36,17 +35,6 @@ impl RawOptionsApply for RawOptimizationOptions {
     self,
     plugins: &mut Vec<Box<dyn rspack_core::Plugin>>,
   ) -> Result<Self::Options, rspack_error::Error> {
-    if let Some(options) = self.split_chunks {
```

Comment:
this implies that we will no longer use ﻿`old_split_chunks_plugin`.

Reply:
https://www.rspack.dev/blog/announcing-0.4.html#deprecatingbuiltinsdevfriendlysplitchunks-andexperimentsnewsplitchunks

this is a breaking and should move to 0.5.0, we haven't remove oldSplitChunks currently, so user can `newSplitChunks: false` to enforce to use oldSplitChunks

Reply:
it's a mistake we haven't remove it earlier, since that some shit code is needed